### ISSUE TITLE
Add `--version` command to get the current installed version

### ISSUE THREAD

**Issue by ajpinedam** [unoplatform/uno.check#378]
It'd be nice to be able to execute `Uno.Check` with the `--version` flag to know the current version installed.

I know you can get this information using the dotnet command

```bash
> dotnet tool list -g
Package Id      Version      Commands
--------------------------------------
uno.check       1.30.1       uno-check
```

or executing Uno.Check and then cancel the execution.

```bash

           ╓╣╜"╙╬╖
         ╓╣`     "╣╗
       ,@Ñ╜╩╬╗     ╙▒Ñ%,
     ╓╬╝     ]╣╦   ╓▓  ╙╣╖
    ╬╝     ,@╜  ╚@╬╜     ▒
    ╬╗   ,╣╜ ╝@╦╣`     ╓╣╜
     ╙╬╦ ▒     ╙@╖   ╓╣╜
        ╙╝@      "╙╣▒
           ╚h    ,@╝
             ╙Ñ@Ñ╜
 

                                         _                     _
  _   _   _ __     ___             ___  | |__     ___    ___  | | __
 | | | | | '_ \   / _ \   _____   / __| | '_ \   / _ \  / __| | |/ /
 | |_| | | | | | | (_) | |_____| | (__  | | | | |  __/ | (__  |   <
  \__,_| |_| |_|  \___/           \___| |_| |_|  \___|  \___| |_|\_\

 **Uno Platform Check v1.30.1.0**
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
This tool will check your Uno Platform development environment.
If problems are detected, it will offer the option to try and fix them for you, or suggest a way to fix them yourself.
```

but providing a `--version` command would match `Uno.Check` with most of the other CLI utilities.

# Usage

```bash
> uno-check --version
Uno.Check Version 1.30.1 (2025-03-05)
```
